.github/workflows/ci.yaml

```
@@ -51,16 +55,13 @@ jobs:
 
       - name: Run mypy
         run: tox -e typing
-
+  
   test:
     runs-on: ubuntu-latest
 
     strategy:
       matrix:
-        python-version:
-          - 3.7
-          - 3.8
-          - 3.9
+        python-version: ['3.10', '3.11', '3.12']
```

Do you want to throw in Python 3.13 that was just released while you are at it?
```suggestion
        python-version: ['3.10', '3.11', '3.12', '3.13']
```